dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/Connector.java

```diff
@@ -36,10 +38,19 @@ public enum DefaultProperties implements ConnectorProperty {
     public String getName();
 
     @Nonnull
-    public String getDefaultFileName();
+    default String getDefaultFileName() {
```

Comment:
I wonder if this change will break existing CW, translation and Assessment loaders.

E.g. for metadata connectors:

* Batch translation looks for just `.zip` file to locate metadata
* Assessment expects the file to end with a specific prefix, i.e. `redshift-metadata.zip`
* I'm not in the context of CW loaders

I'm not sure about query logs connectors (I believe there are only Assessment and CW loaders there).

Reply:
The change of the filenames needs to be applied only when the `--assessment` flag is used in order not to break other tools.

Reply:
Added the `--assessment` flag check. Keeping in draft mode until the loaders are ready for this